pkg/credentials/file_store_test.go

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package credentials
+
+import (
+	"context"
+	"testing"
+
+	"github.com/vmware-tanzu/velero/pkg/builder"
+
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+
+	velerotest "github.com/vmware-tanzu/velero/pkg/test"
+)
+
+func TestNamespacedFileStore(t *testing.T) {
+	testCases := []struct {
+		name             string
+		namespace        string
+		fsRoot           string
+		secrets          []*corev1.Secret
+		secretSelector   *corev1.SecretKeySelector
+		wantErr          string
+		expectedPath     string
+		expectedContents string
+	}{
+		{
+			name:           "Get returns an error if the secret can't be found",
```

Comment:
Please try to avoid having any variable or method name (in this case, `Get`) in test case naming. We shouldn't need to change the name of a test case when we rename/refactor things.